src/Client-SDK/Temporal.WorkflowClient/ClientApi.cs

```diff
@@ -0,0 +1,466 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Temporal.Common.DataModel;
+using Temporal.Common.WorkflowConfiguration;
+
+using Temporal.Async;
+using Temporal.Collections;
+
+namespace Temporal.WorkflowClient
+{
+    public class ClientApi
+    {
+    }
+
+    public class TemporalServiceClient
```

Comment:
+1 - though I would not discount providing a more user-friendly wrapper over the client depending on how nice it is to use in C#. However, I think for now we should just use the protobuf one and consider this more later to avoid doing too much up front.

Reply:
IO am not opposed to this. However, I found it quite confusing when learning the Java SDK: it was not clear how we decided which APIs were wrapped and which were not. The goal here is to wrap all APIs that are "interesting" for a client user. There are only a few of them, so it is not much effort. 

Either way, it is not the most critical detail at the current (early) stage.